mylist.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'myenumerable'
```

Comment:
- Kindly change the file name that you're requiring above ☝️ to `my_enumerable` as that is the correct file name.
I am currently getting an error when I try to use the Interactive Ruby Shell(IRB).

Here 👇 is a screenshot of what I got when I use the IRB

![](https://i.imgur.com/YhgLYCt.png)

Reply:
Thank you for the tip. I have fixed the issue